src/blockchain.py

```diff
@@ -50,6 +52,8 @@ def __init__(self, override_constants: Dict = {}):
         self.height_to_hash: Dict[uint32, bytes32] = {}
         # All headers (but not orphans) from genesis to the tip are guaranteed to be in header_blocks
         self.header_blocks: Dict[bytes32, HeaderBlock] = {}
+        cpu_count = multiprocessing.cpu_count()
+        self.pool = multiprocessing.Pool(max(cpu_count - 1, 1))
```

Comment:
Maybe increase this to cpu_count, to take more advantage of dual core CPUs?

Reply:
I was thinking to leave one core for just full_node, but maybe it makes more not to do that.  
You probably have a better feel on how much other stuff full node is doing and cpu needs

Reply:
During sync, it's not doing almost anything. 
After sync, it will be very busy with transactions and signature validation. 
However maybe a dual core cpu user still wants to use their computer normally?

Reply:
just checked cpu_count will return 2x physical core count in hyperthreading is enabled. 
The way it's now 3/4 will be used for validation 1/4 for full node.
If we want to reduce footprint we could make validators + fullnode = cpu_count / 2
It's probaly worth making this configurable by user ?